lib/ansible/modules/network/aci/aci_interface_policy_leaf_policy_group.py

```diff
@@ -0,0 +1,395 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_interface_policy_leaf_policy_group
+short_description: Add Fabric Interface Policy Leaf Policy Groups on Cisco ACI fabrics.
+description:
+- Add Fabric Interface Policy Leaf Policy Groups on Cisco ACI fabrics.
+- More information from the internal APIC class I(infra:AccBndlGrp), I(infra:AccPortGrp) at
+  U(https://developer.cisco.com/site/aci/docs/apis/apic-mim-ref/).
+author:
+- Bruno Calogero (@brunocalogero)
+version_added: '2.5'
+notes:
+- When using the module please select the appropriate link_aggregation_type. 'link'(PC)/'node'(VPC), 'leaf'(Leaf Access Port Policy Group)
+options:
+ policy_group:
+   description:
+   - Name of the leaf policy group to be added/deleted.
+   aliases: [ name, policy_group_name ]
+ description:
+   description:
+   - Description for the leaf policy group to be created.
+   aliases: [ descr, description ]
+ link_aggregation_type:
+   description:
+   - Selector for the type of leaf policy group we want to create.
+   aliases: [ link_aggregation_type_name ]
```

Comment:
I wonder if this wouldn't be better named `lag_type` either as the default name, or an alias. As LAG seems to be a common term (at least I know this term).
cc @rsmeyers @jmcgill298

Reply:
agreed, was just wondering if we should make it same as GUI or not 👍